### ISSUE TITLE
failed to solve: failed to read dockerfile

### ISSUE THREAD

**Issue by thijsjek** [0x25CBFC4F/betternote#2]
```
[+] Building 0.1s (2/2) FINISHED
 => [betternote internal] load .dockerignore                                                                                                                                                                                                                                                                            0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                         0.0s
 => [betternote internal] load build definition from Dockerfile                                                                                                                                                                                                                                                         0.0s
 => => transferring dockerfile: 2B                                                                                                                                                                                                                                                                                      0.0s
failed to solve: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount1063389014/Dockerfile: no such file or directory
```

after copying your docker-compose, and starting the container:
```
$ docker compose start
service "redis" has no container to start
```

Client: Docker Engine - Community
 Version:           24.0.5
 API version:       1.43
 Go version:        go1.20.6
 Git commit:        ced0996
 Built:             Fri Jul 21 20:35:45 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.5
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.6
  Git commit:       a61e2b4
  Built:            Fri Jul 21 20:35:45 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.22
  GitCommit:        8165feabfdfe38c65b599c4993d227328c231fca
 runc:
  Version:          1.1.8
  GitCommit:        v1.1.8-0-g82f18fe
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

**Comment by 0x25CBFC4F** [comment]
Hey! Thank you for trying my project out.

~~This docker-compose requires build and a cloned git project.~~

~~Try this out:~~

```bash
git clone https://github.com/0x25CBFC4F/betternote.git
cd betternote
docker compose pull
docker compose build
docker compose up -d
```

**Comment by 0x25CBFC4F** [comment]
I just merged #3.
You no longer need to build docker-compose.
Pre-compiled docker images are now available.
Please use `docker compose up -d`, not `docker compose start`

**Comment by 0x25CBFC4F** [comment]
Closing this issue for now, @thijsjek please notify me if you have any more questions.
